docs/src/modules/sandbox/CodeSandbox.ts

```diff
@@ -47,6 +47,19 @@ function createReactApp(demoData: DemoData) {
     [`src/Demo.${ext}`]: {
       content: demoData.raw,
     },
+    // Spread the relative modules
+    ...(demoData.relativeModules &&
+      // Transform the relative modules array into an object
+      demoData.relativeModules.reduce(
+        (prev, curr) => ({
+          ...prev,
```

Comment:
Usually called accumulator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce#examples, shorten `acc` in most of the codebase:

```suggestion
        (acc, curr) => ({
          ...acc,
```